### PR TITLE
[PE-4477] Added title attribute to Expansion panel

### PR DESCRIPTION
**Pull request by janibolkvadze**
- [x] Added support of attribute Title
- [x] Added Cypress test to check proper functionality of the attribute

@dani-cl-madrid @mattnickles @jllr 

https://ctl.atlassian.net/browse/PE-4477

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://0794b6aee2103d56be9882c2cb75f722afbc04a3-chi-pr.enterprise.ctl.io" target="_blank"> https://0794b6aee2103d56be9882c2cb75f722afbc04a3-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/727/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://0c16a6c00f4957f22f6eacaf206d2242c2e8bd0f-chi-pr.enterprise.ctl.io" target="_blank"> https://0c16a6c00f4957f22f6eacaf206d2242c2e8bd0f-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/740/ for more details.

File: src/custom-elements/src/components/expansion-panel/expansion-panel.tsx [hunk 1]
```diff
@@ -38,6 +39,26 @@ export class ExpansionPanel {
...
+    };
+
+    const subscriberCallback = (mutations) => {
+      mutations.forEach((mutation) => {
+        this.epanelTitle = mutation.target.title;
+      });
+    }
+
+    const observer = new MutationObserver(subscriberCallback);
+    observer.observe(target, config);
```
Comment (dani-cl-madrid) [inline src/custom-elements/src/components/expansion-panel/expansion-panel.tsx]: Don't forget to call disconnect.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://9422c4b65a9fad20d1dfb69441455914cc4ad543-chi-pr.enterprise.ctl.io" target="_blank"> https://9422c4b65a9fad20d1dfb69441455914cc4ad543-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/749/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://05624510a29b6f0f7e87fb1ceddb10d658d296e1-chi-pr.enterprise.ctl.io" target="_blank"> https://05624510a29b6f0f7e87fb1ceddb10d658d296e1-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/754/ for more details.

**Comment by jllr** [comment]
@janibolkvadze please solve conflicts

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://8708298813186161aac57d5be667bf25b447037b-chi-pr.enterprise.ctl.io" target="_blank"> https://8708298813186161aac57d5be667bf25b447037b-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/782/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://7b18758529ff27bd13209a9284ad44121306e928-chi-pr.enterprise.ctl.io" target="_blank"> https://7b18758529ff27bd13209a9284ad44121306e928-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/783/ for more details.

**Comment by jllr** [comment]
@dani-cl-madrid can you take a look and review it? Thanks in advance

**Comment by janibolkvadze** [comment]
> I really like this approach to be able to use title variable. Congratulations Jani!

@dani-cl-madrid Thanks for your help!

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://db284839205b8464201e394cc73a5e1eb856c635-chi-pr.enterprise.ctl.io" target="_blank"> https://db284839205b8464201e394cc73a5e1eb856c635-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/787/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-0tfxhn, https://ops.elasticbox.com/services/instances/i-1wo8w5, https://ops.elasticbox.com/services/instances/i-mkg4u7, https://ops.elasticbox.com/services/instances/i-5wskqp, https://ops.elasticbox.com/services/instances/i-41iha7, https://ops.elasticbox.com/services/instances/i-g8ynj7, https://ops.elasticbox.com/services/instances/i-fm9tm0

**Pull request merged**